pkg/reconciler/openshift/tektonaddon/extension.go

```diff
@@ -120,21 +117,25 @@ func (oe openshiftExtension) PostReconcile(ctx context.Context, comp v1alpha1.Te
 			}
 
 			if err := createInstallerSet(ctx, oe.operatorClientSet, addon, manifest, oe.version,
-				miscellaneousResourcesInstallerSet, "addon-openshift"); err != nil {
+				MiscellaneousResourcesInstallerSet, "addon-openshift"); err != nil {
 				return err
 			}
 		}
 		logger.Error("failed to get InstallerSet: %s", err)
 		return err
 	}
 
+	if len(installedTIS.Items) == 0 {
+		return v1alpha1.RECONCILE_AGAIN_ERR
+	}
```

Comment:
as creating a new installerset will not have any effect on the length of current installedTIS.items, let us move this return to https://github.com/tektoncd/operator/pull/574/files#r782647382